packages/overlays/addon/components/drawer/index.ts

```diff
@@ -67,6 +67,12 @@ export default class Drawer extends Component<DrawerArgs> {
     }
   }
 
+  @action handleOpen(): void {
```

Comment:
We can eliminate this action when passing the argument directly to the Overlay component.